ext/pdo/php_pdo_driver.h

```diff
@@ -558,6 +558,8 @@ struct _pdo_stmt_t {
 	/* if true, we've already successfully executed this statement at least
 	 * once */
 	unsigned executed:1;
+	/* if true, it means select all fields */
+	unsigned select_all:1;
```

Comment:
This would increase the size of the bitfield, and to counteract that, `unsigned _reserved:29;` a few lines below would need to be changed to `unsigned _reserved:28;`